deployments/helm/nsm/charts/spire/registration.json

```diff
@@ -59,6 +79,36 @@
       ],
       "spiffe_id": "spiffe://test.com/nsmrs",
       "parent_id": "spiffe://test.com/spire-agent"
+    },
+    {
+      "selectors": [
+        {
+          "type": "k8s",
+          "value": "sa:example-connectivity-domain-service-account"
+        }
+      ],
+      "spiffe_id": "spiffe://test.com/example-connectivity-domain",
+      "parent_id": "spiffe://test.com/spire-agent"
+    },
+    {
+      "selectors": [
+        {
+          "type": "k8s",
+          "value": "sa:example-connectivity-domain-app-dns-publisher-service-account"
```

Comment:
Just a curiosity, is this info added in the member core operator separately from this file, right? And is this file used to deploy spire in central cluster? 
We can not have these values with 'example' configurable since the file is not .tpl, right?

Reply:
> is this info added in the member core operator separately from this file, right?

No, the membercore is only aware of the workloads in the member clusters. It doesn't need the know about central cluster workloads. 

> And is this file used to deploy spire in central cluster?

This file is used only when we deploy spire into the central cluster. 

> We can not have these values with 'example' configurable since the file is not .tpl, right?

Yes we can not have them like that but we can modify the implementation in the following two ways:
1. Generate the ``registration.json`` and then use it.  (as mentioned bellow)
2. We can get rid of the ``registration.json`` and create a ``configmap``